content/goldcard-holders-faq/upcoming-changes.md

```diff
@@ -1,11 +1,44 @@
 ---
 title: "Upcoming Changes"
-draft: true
+draft: false
 weight: 14
 ---
 
-There is currently some new [legislative push to improve the Gold Card](https://www.ndc.gov.tw/en/Content_List.aspx?n=999F9864EFDB5F6F&upn=6CE244D6E7DAF831), lead by the National Development Council:
+On June 18th 2021, a major amendment to the law underpinning the Gold Card
+ was passed by the legislature. It's since been signed into law by the President
+and is currently being implemented by the various government department responsible.
+
+At lot of things are still unclear.
+This page contains fully unofficial information based on what we know about the law,
+please confirm with relevant authorities before acting!
+
+## When can I get the good stuff from the new law?
+October 2021.
+
+## I can get permanent residence (APRC) after 3 years now?
+Yes, if:
+* it's been more than 3 years (to the day) since you entered Taiwan on your Gold Card
```

Comment:
@fifieldt is it 3 years since entered Taiwan or 3 years since the Gold Card was first issued?

Reply:
I believe it's both.